CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Developer Changelog
 
+### April 8, 2022 DESCW-282
```

Comment:
could probably wrap these bullets under the previous header, but that is not strictly necessary in my mind, as long as this information is captured.